keyboards/ymdk_bface/info.json

```
@@ -0,0 +1,20 @@
+{
+  "keyboard_name": "YMDK Bface",
+  "url": "",
+  "maintainer": "qmk",
+  "width": 15,
+  "height": 5,
+  "layouts": {
+      "LAYOUT_60_all": {
```

This should just be `LAYOUT_all`.

```suggestion
      "LAYOUT_all": {
```